### PR TITLE
libyafaray: 3.5.1 -> unstable-2022-09-17

### PR DESCRIPTION
**Pull request by superherointj**
* libyafaray 3.5.1 -> unstable-2022-09-17

Upstream renamed `yafaray-core` to `libyafaray`.

### CODE REVIEW

**Comment by wegank** [comment]
Interestingly, it actually builds well on aarch64-linux...

**Comment by superherointj** [comment]
> Interestingly, it actually builds well on aarch64-linux...

My understanding of aarch64-linux being unsupported came from the platforms definition:
image

Will try to build and update accordingly.

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 192645` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

File: pkgs/tools/graphics/yafaray-core/default.nix [hunk 1]
```diff
@@ -31,7 +31,8 @@ stdenv.mkDerivation rec {
       homepage = "http://www.yafaray.org";
       maintainers = with maintainers; [ hodapp ];
       license = licenses.lgpl21;
-      platforms = [ "x86_64-linux" ];
+      platforms = [ "x86_64-linux" "aarch64-linux" ];
+      broken = !stdenv.isAarch64 && stdenv.isLinux; # at 2022-09-23
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/yafaray-core/default.nix:35]: Please add a comment with a short excerpt from the error like 'linking fails: undefined reference to ...'

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 192645` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>yafaray-core</li>
  </ul>
</details>

File: pkgs/tools/graphics/yafaray-core/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/tools/graphics/yafaray-core/default.nix]: Done

**Comment by wegank** [comment]
Maybe we could bump this package to 

```nix
{
  version = "unstable-2022-09-17";

  src = fetchFromGitHub {
    owner  = "YafaRay";
    repo   = "libYafaRay";
    rev    = "6e8c45fb150185b3356220e5f99478f20408ee49";
    sha256 = "sha256-UVBA1vXOuLg4RT+BdF4rhbZ6I9ySeZX0N81gh3MH84I=";
  };
}
```

**Review by wegank — APPROVED** [review]

**Comment by superherointj** [comment]
> Maybe we could bump this package to
> 
> ```nix
> {
>   version = "unstable-2022-09-17";
> 
>   src = fetchFromGitHub {
>     owner  = "YafaRay";
>     repo   = "libYafaRay";
>     rev    = "6e8c45fb150185b3356220e5f99478f20408ee49";
>     sha256 = "sha256-UVBA1vXOuLg4RT+BdF4rhbZ6I9ySeZX0N81gh3MH84I=";
>   };
> }
> ```
> 
> so that it can be built with GCC 11.



> Maybe we could bump this package to
> 
> ```nix
> {
>   version = "unstable-2022-09-17";
> 
>   src = fetchFromGitHub {
>     owner  = "YafaRay";
>     repo   = "libYafaRay";
>     rev    = "6e8c45fb150185b3356220e5f99478f20408ee49";
>     sha256 = "sha256-UVBA1vXOuLg4RT+BdF4rhbZ6I9ySeZX0N81gh3MH84I=";
>   };
> }
> ```
> 
> so that it can be built with GCC 11.

It builds fine. Updated PR. Please review again.

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 192645` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>yafaray-core</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>libyafaray</li>
  </ul>
</details>

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 192645` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>yafaray-core</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>libyafaray</li>
  </ul>
</details>

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 192645` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>yafaray-core</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>libyafaray</li>
  </ul>
</details>

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 192645` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>yafaray-core</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>libyafaray</li>
  </ul>
</details>

**Review by wegank — APPROVED** [review]

**Pull request merged**